ajax/libs/academicons/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "academicons",
+  "filename": "css/academicons.min.css",
+  "description": "Academicons is an icon font for academics. It aims to supplement other packages (such as Font Awesome) with icons for academic websites.",
+  "homepage": "https://jpswalsh.github.io/academicons/",
+  "keywords": [
+    "css",
+    "font",
+    "icons",
+    "academic",
+    "research",
+    "university"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jpswalsh/academicons.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/academicons*",
+          "fonts/academicons*"
```

Comment:
We can make it easier. Use `"+(css|fonts)/academicons*"`instead of these two lines.